src/app/pages/common/entity/entity-table/entity-table.component.ts

```diff
@@ -592,7 +596,7 @@ export class EntityTableComponent<Row = any> implements OnInit, AfterViewChecked
       },
       (res: any) => {
         this.isTableEmpty = true;
-        this.configureEmptyTable(EmptyType.Errors, res);
+        this.configureEmptyTable(EmptyType.Errors, res.reason);
```

Comment:
I don't think res is guaranteed to have `reason` as evidenced by code several lines below.
Let's do `res.reason || res.error` or something similar.

Reply:
Also please change `error: any` in `configureEmptyTable` to `error: string`.